Example/GiniVision/ScreenAPIViewController.swift

```diff
@@ -99,6 +99,7 @@ class ScreenAPIViewController: UIViewController {
         
         cancelAnalsyis()
         imageData = data
+        print("Analysing document with size \(Double(data.count) / 1024.0)")
```

Comment:
Oh, I just realised that this log message is not in the library, but in the example app. So some additional logging shouldn't be a problem.